android/app/src/main/java/com/on/turip/ui/compose/setting/SettingScreen.kt

```diff
@@ -0,0 +1,272 @@
+    onClickPrivacyPolicy: () -> Unit,
+    onClickLogin: () -> Unit,
+    onClickLogout: () -> Unit,
+    onClickWithdraw: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier.fillMaxWidth(),
+        verticalArrangement = Arrangement.spacedBy(5.dp),
+    ) {
+        SettingCommonScreen(onClickInquiry, onClickPrivacyPolicy)
+
+        when (uiState.memberStatus) {
+            MemberStatus.MEMBER -> {
+                SettingForMemberScreen(onClickLogout, onClickWithdraw)
+            }
+
+            MemberStatus.GUEST -> {
+                SettingForGuestScreen(onClickLogin)
+            }
+        }
+    }
+}
+
+@Composable
+private fun SettingCommonScreen(
+    onClickInquiry: () -> Unit,
+    onClickPrivacyPolicy: () -> Unit,
+) {
+    HorizontalDivider(
+        thickness = 8.dp,
+        color = colorResource(R.color.gray_100_f0f0ee),
+    )
+
+    SettingItem(
+        uiModel =
+            SettingModel(
+                iconResource = R.drawable.ic_inquire,
+                titleResource = R.string.setting_inquiry,
+            ),
+        onClick = onClickInquiry,
+        modifier = Modifier.fillMaxWidth(),
+    )
+    SettingItem(
+        uiModel =
+            SettingModel(
+                iconResource = R.drawable.ic_document,
+                titleResource = R.string.setting_privacy_policy,
+            ),
+        onClick = onClickPrivacyPolicy,
+        modifier = Modifier.fillMaxWidth(),
+    )
+}
+
+@Composable
+private fun SettingForGuestScreen(onClickLogin: () -> Unit) {
+    SettingItem(
+        uiModel =
+            SettingModel(
+                iconResource = R.drawable.ic_login,
+                titleResource = R.string.setting_login,
+            ),
+        onClick = onClickLogin,
+        modifier = Modifier.fillMaxWidth(),
+    )
+}
+
+@Composable
+private fun SettingForMemberScreen(
+    onClickLogout: () -> Unit,
+    onClickWithdraw: () -> Unit,
+) {
+    SettingItem(
+        uiModel =
+            SettingModel(
+                iconResource = R.drawable.ic_logout,
+                titleResource = R.string.setting_logout,
+            ),
+        onClick = onClickLogout,
+        modifier = Modifier.fillMaxWidth(),
+    )
+    SettingItem(
+        uiModel =
+            SettingModel(
+                iconResource = R.drawable.ic_withdraw,
+                titleResource = R.string.setting_withdraw,
+            ),
+        onClick = onClickWithdraw,
+        modifier = Modifier.fillMaxWidth(),
+    )
+}
+
+@Preview(showBackground = true, name = "멤버")
+@Composable
+private fun MemberSettingScreenPreview() {
+    TuripTheme {
+        SettingScreenContent(
+            uiState =
+                SettingUiState(
+                    deviceIdentifier = TuripDeviceIdentifier.EMPTY,
+                    memberStatus = MemberStatus.MEMBER,
+                    showLogoutDialog = false,
+                    showWithdrawDialog = false,
+                ),
```

Comment:
```suggestion
            uiState =
                SettingUiState.EMPTY,
```

c: 이렇게 써도 될 것 같네여 하지만, EMPTY 상태에서 기본값이 MEMBER인 건 조금 어색해서,
MemberStatus에 UNDEFINED 같은 기본 상태를 하나 추가해두는 게 더 안전하고 명확해 보이네여😁

Reply:
여기는 Member 일 때 화면을 확인하기 위한 `Preview` 에요 !

SettingUiState.EMPTY 에서 기본값이 멤버로 임시 구현되어 있는 부분은 로그인 API 합쳐질 때 NONE 타입으로 수정이 필요할 것 같아요!
MemberStatus 파일은 로그인 API 구현할 때 정의한 Member 타입 사용하도록 수정하면서 제거 될 것 같아요!